Assets/Scripts/Layers/Properties/PropertyData/Asset.cs

```diff
@@ -0,0 +1,15 @@
+namespace Netherlands3D.Twin.Layers.Properties
+{
+    public struct Asset
+    {
+        /// <summary>
+        /// The reference string that is used as an identifier in the PropertyData to refer to this asset
+        /// </summary>
+        public string Reference;
```

Comment:
Is dit de vervanging voor prefabID?

Reply:
Nee, een Asset is een referentie aan de CSV of OBJ (of andere asset) die in de zipfile moet worden opgeslagen. Door in de propertydata de referentie naar de asset file op te nemen kan je makkelijk door het verwijderen van een laag voorkomen dat het bestand de volgende keer in de zip file komt.